lib/diagnostic.rb

```diff
@@ -78,12 +99,13 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = I am a HouseCat, and I go 'meow'
 
 ## Explain why this would be the output, based on the method lookup chain.
 # your response as a comment here
 ##
-
+# This creates a new HouseCat, since it inherits from Cat which inherits from Animal,
```

Comment:
Correct, but we want a mention of why `say_hello` works due to method lookups on superclasses.